src/layout/Header/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './style.css';
+import logo from "../../../public/images/logo.png";
```

Comment:
Did you consider exporting your images from an index.js file in your images folder to make your file paths a bit shorter?